pynumaflow/function/server.py

```diff
@@ -152,8 +152,8 @@ def ReduceFn(
                 f"got key: {key}, start: {start}, end: {end}."
             )
 
-        start_dt = datetime.fromtimestamp(int(start), timezone.utc)
-        end_dt = datetime.fromtimestamp(int(end), timezone.utc)
+        start_dt = datetime.fromtimestamp(int(start) / 1000, timezone.utc)
```

Comment:
@ab93 we cannot support milliseconds in python?

Reply:
It is milliseconds.

```sh
>>> print(datetime.fromtimestamp(1661169600000/1000, timezone.utc))
2022-08-22 12:00:00+00:00
>>> print(datetime.fromtimestamp(1661169600060/1000, timezone.utc))
2022-08-22 12:00:00.060000+00:00
```

Reply:
This is the preferred way from what I can understand. We don't lose granularity because it converts to float.

Reply:
```suggestion
        start_dt = datetime.fromtimestamp(int(start) / 1e3, timezone.utc)
```